src/consentManagement/createConsentManagement.ts

```diff
@@ -98,7 +106,8 @@ export function createConsentManagement<
         useConsent,
         ConsentBannerAndConsentManagement,
         FooterConsentManagementItem,
-        FooterPersonalDataPolicyItem
+        FooterPersonalDataPolicyItem,
+        consentLocalStorageKey: localStorageKey
```

Comment:
```suggestion
        FooterPersonalDataPolicyItem,
        "consentLocalStorageKey": localStorageKey
```  

Will you actually use this value?  
Because if there is no identified usecase for it no need to increace the API surface.

Reply:
Yeah so I see, you exported it for Storybook.  
The storybook needs shouldn't affect the API.

Reply:
Ok after thinking, the use case we have won't be covered by just returning the computed key. So, I removed it from the return.